src/screens/About/About.tsx

```diff
@@ -11,19 +11,30 @@ import TeamTable from './TeamTable/TeamTable';
 import HeadshotGrid, { HeadshotGrid2Up } from './HeadshotGrid/HeadshotGrid';
 import { PartnerLogoGrid } from 'components/LogoGrid/LogoGrid';
 
-import { Wrapper, Content, Header } from './About.style';
+import {
+  Wrapper,
+  Content,
+  Header,
+  ActiveAlumniButtonContainer,
+  ActiveAlumniButton,
+} from './About.style';
 
 const sidebar = (
   <React.Fragment>
-    <SidebarLink href="#can">Covid Act Now</SidebarLink>
-    <SidebarLink href="#partners">Our Partners</SidebarLink>
-    <SidebarLink href="#team">The Team</SidebarLink>
-    <SidebarLink href="#model">The Model</SidebarLink>
+    <SidebarLink href="#can">About Covid Act Now</SidebarLink>
+    <SidebarLink href="#partners">Partners</SidebarLink>
+    <SidebarLink href="#founders">Founders</SidebarLink>
+    <SidebarLink href="#advisors">Advisors</SidebarLink>
+    <SidebarLink href="#team">Team</SidebarLink>
+    <SidebarLink href="#model">Model Info</SidebarLink>
```

Comment:
nit: here it says "Model Info" but the actual heading for that section is "Our Model".  Intentional?

Reply:
hmm good catch. they were that way in the wireframe but ill just sync them